### ISSUE TITLE
Restore Layout makes maximised main window smaller (windowed)

### ISSUE THREAD

**Issue by tlaemmlein** [KDAB/KDDockWidgets#287]
Hi,
System: Windows 10 / VS 2019
kddockwidgets version: 1.6.0 with bug fix: https://github.com/KDAB/KDDockWidgets/issues/96

Steps:
* Open kddockwigets_example
* Maximise the main window
* Click Save layout
* Click Restore layout
* Observe the maximised window is smaller (windowed)

But it should stay unchanged.

Regards,
Thomas

**Comment by iamsergio** [comment]
Is it related to the #96 fix ? I'm not on Windows right now to test

**Comment by tlaemmlein** [comment]
Yes. I suppose too that it blongs to #96 .  And the problem occurs on windows.
